tests/common.go

```diff
@@ -2763,6 +2763,55 @@ func DeleteBackupLocation(name string, backupLocationUID string, orgID string) e
 
 }
 
+// DeleteScheduleBackup Delete Schedule Backup
+func DeleteSchedule(backupScheduleName, backupScheduleUID, schedulePolicyName, schedulePolicyUID, OrgID string) error {
```

Comment:
nit: Follow comment standards, rename function to `DeleteScheduleBackup` or change comment, I think if you are deleting backup schedule here, you should rename func to `DeleteScheduleBackup` rather than changing comment

Reply:
@vpinisetti-px pointed out that `DeleteScheduleBackup` is confusing because it indicates that we are deleting the schedule backups not backup schedule